pkg/sqlite/load.go

```diff
@@ -747,3 +753,27 @@ func (s *SQLLoader) AddBundlePackageChannels(manifest registry.PackageManifest,
 
 	return tx.Commit()
 }
+
+func (s *SQLLoader) addDependencies(tx *sql.Tx, bundle *registry.Bundle) error {
```

Comment:
This should also parse the CSV (if present) and find required apis (if present) and add them as dependencies